Editor/Windows/AssetPaletteProperties.cs

```diff
@@ -203,7 +192,7 @@ public SerializedProperty SelectedFolderSerializedProperty
                 // while on another folder are meant to apply to newly selected folders too.
                 CurrentCollectionSerializedObject.Update();
                 entryPanel.SortEntriesInSerializedObject();
-                CurrentCollectionSerializedObject.ApplyModifiedPropertiesWithoutUndo();
```

Comment:
Careful, this used to specifically not make an undo entry, and now it does. That should probably be a parameter.